pyproject.toml

```diff
@@ -76,6 +76,7 @@ ruamel-yaml = "^0.18.5"
 pymatreader = "^0.0.32"
 xmltodict = "*"
 boto3 = "^1.34.122"
+aiohttp = "^3.7.4"
```

Comment:
Since `check_load_balancer()` has been removed, it seems unnecessary here as well.